packages/components/organisms/f-offers/src/components/SearchBox.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <div :class="$style['c-offers-searchBox']">
+        <search-box
+            class="l-boxWrapper"
+            :config="{
+                queryString: {
+                    refine: 'with_discounts'
+                },
+                isShellHidden: true
+            }"
+            data-test-id="OffersInbox-Searchbox"
+            @address-search-focus="addressFocus"
+            @searchbox-error="handleSearchBoxError"
+            @submit-saved-address="validSavedAddress"
+            @submit-valid-address="validSearch" />
+    </div>
+</template>
+
+<script>
+import { mapActions } from 'vuex';
+import SearchBox from '@justeat/f-searchbox';
+import {
+    ACTION_ADDRESS_FOCUS,
+    ACTION_VALID_SAVED_ADDRESS_SEARCH,
+    ACTION_VALID_SEARCH,
+    ACTION_INVALID_ADDRESS,
+    ACTION_EMPTY_ADDRESS,
+    ACTION_UNKNOWN_ERROR
+} from '../store/types';
+import { POSTCODE_EMPTY, POSTCODE_INVALID } from '../constants';
+
+export default {
+    name: 'OffersSearchBox',
+
+    components: {
+        SearchBox
+    },
+
```

Comment:
Worth registering the offersHeaderSearch module here within a beforeCreate [similarly to how it's done in `f-searchbox`](https://github.com/justeat/fozzie-components/blob/master/packages/components/molecules/f-searchbox/src/components/Form.vue#L280-L284)?

Reply:
yes your right my bad, thanks